packages/contracts-ops/contracts/scripts/DeployImplementations.s.sol

```diff
@@ -88,7 +88,6 @@ contract DeployImplementations is DeployImplementationsLogic {
         vm.startBroadcast();
         deployImplementations(_domain);
         write(buffer, indent.nextIndent(), _domain);
-        vm.stopBroadcast();
```

Comment:
reasoning?

Reply:
foundry quirk (read: bug) - calling vm`.startBroadcast()` again in the parent script was reverting, whereas if you just never `stopBroadcast()` then it's fine